src/panels/config/automation/action/types/ha-automation-action-device_id.ts

```diff
@@ -51,7 +51,7 @@ export class HaDeviceAction extends LitElement {
   );
 
   protected render() {
-    const deviceId = this._deviceId || this.action.device_id;
```

Comment:
`deviceId` will now only be defined when it is picked in the UI. When you open an existing automation it will not be set.